src/Models/SprintStory.php

```
@@ -120,4 +120,48 @@ public static function getSprintLoad(Database $db, int $sprintId): int
+            [':sprint_id' => $sprintId]
+        );
+        return $stmt->fetchAll();
+    }
+
+    /**
+     * Update the board-level status of a story within a sprint.
+     * Separate from user_stories.status (spec MR decision: keep them independent).
+     *
+     * @param string $status One of 'backlog', 'in_progress', 'done'
+     */
+    public static function updateBoardStatus(Database $db, int $sprintId, int $storyId, string $status): void
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)